src/main/containers/SourcePanel.jsx

```diff
@@ -12,7 +12,7 @@ const SourcePanel = () => {
         Request bar
         Data canvas
       */}
-    <UrlBar SourceOrDest="source" requestBody='' />
+    <RequestBar SourceOrDest="source" requestBody='' />
```

Comment:
Let's keep things consistent and use single quotes here. The linting rules should have caught this, let me know if there is something up with your linter I'll take a peak.